tests/test1.rs

```diff
@@ -40,5 +40,6 @@ fn is_not() {
 #[test]
 fn exported_public_method_defined_by_macro() {
   let a = &b"ab12cd\nefgh"[..];
-  assert_eq!(not_line_ending(a), IResult::Done(&b"\nefgh"[..], &b"ab12cd"[..]));
+  let expected: IResult<_, _> = IResult::Done(&b"\nefgh"[..], &b"ab12cd"[..]);
```

Comment:
default parameter does not works for function, so

return types for codes that calls macro directly from assert_eq() cannot be inferred.

```rust

```

In real world usecase, it would not affect any usage, unless users write code like

```rust
assert_eq!(macro_in_nom!("args.."), Done(...))
```